atlas/src/views/SearchView.tsx

```diff
@@ -38,8 +38,11 @@ const SearchView: React.FC<SearchViewProps> = ({ search = '' }) => {
   if (loading || !data) {
     return <p>Loading...</p>
   }
+  if (error) {
+    throw error
+  }
   if (!data.search) {
-    return <p>Something went wrong...</p>
+    return <p>There was a problem loading your search...</p>
```

Comment:
If `data.search` is null, something went wrong so we shouldn't handle that case separately - from the user's point of view this should be the same case as when `error` is defined

Reply:
Agree. Changed this to throw here as well